.github/workflows/nix.yml

```diff
@@ -24,7 +24,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['default']
```

Comment:
Branch protection rules need to be updated

Reply:
Done